manifests/0000_10_kube-apiserver-operator_02_service.yaml

```diff
@@ -2,9 +2,9 @@ apiVersion: v1
 kind: Service
 metadata:
   annotations:
-    service.alpha.openshift.io/serving-cert-secret-name: openshift-kube-apiserver-operator-serving-cert
+    service.alpha.openshift.io/serving-cert-secret-name: kube-apiserver-operator-serving-cert
   labels:
-    app: openshift-kube-apiserver-operator
+    app: kube-apiserver-operator
```

Comment:
when will we rename the repos?

Reply:
> when will we rename the repos?

Never.  Clayton prefers `cluster-` in the binary and I don't care enough to face the image pain

Reply:
Yes, please!